visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts

```diff
@@ -397,13 +397,21 @@ describe("codeMapMouseEventService", () => {
 				expect(threeSceneService.selectBuilding).toHaveBeenCalledWith(codeMapBuilding)
 			})
 
-			it("should deselect building, when nothing is highlighted and something is selected", () => {
-				threeSceneService.getHighlightedBuilding = jest.fn()
+			it("should call clearselection, when the mouse has not moved while left button was pressed", () => {
+				threeSceneService["selected"] = codeMapBuilding
+				codeMapMouseEventService["hasMouseMoved"] = jest.fn().mockReturnValue(false)
 
 				codeMapMouseEventService.onDocumentMouseUp(event)
 
 				expect(threeSceneService.clearSelection).toHaveBeenCalled()
 			})
+			it("should not call clear selection, whe mouse has moved while left button was pressed", () => {
```

Comment:
- add empty line break between tests
- fix typo `whe mouse has moved` to `when mouse has moved`